portfolio/src/main/webapp/test.html

```diff
@@ -2,22 +2,36 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Contact | My Portfolio</title>
+    <title>My Portfolio</title>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" 
         integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <script src="test.js"></script>
```

Comment:
Let's change the name of this file. test.js isn't a very informative name. What are you trying to do with test.js?

Reply:
Fixed. Will try using more descriptive names in future.